### PR TITLE
refactor(website): separate header from example layout

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Make it possible to use the new header without having code sandboxes, guidelines, props tables, etc.

### Potential Breaking Changes

It might break the web scraping configuration of our sitemap source, but I will check it out once its merged.

### CODE REVIEW

**Review by aiheon — APPROVED** [review]

File: packages/website/src/building-blocs/PageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    componentSourcePath,
+}) => (
+    <div className="plasma-page-header">
+        <h2 className="h5-subdued normal-white-space">{section}</h2>
+        <h1 className="h1-light normal-white-space" data-coveo-field="title">
+            {title}
+        </h1>
+        <h3 className="h4-book-subdued" data-coveo-field="description">
+            {description}
+        </h3>
```
Comment (toofff) [inline packages/website/src/building-blocs/PageHeader.tsx:33]: There are small issues with the title architecture

**Tab Implementation**
image

**Tab Guidelines**
image

**Review by toofff — APPROVED** [review]
Apart from the problem with the title architecture, the rest is OK

File: packages/website/src/building-blocs/PageHeader.tsx (hunk 1 above)
Comment (gdostie) [reply packages/website/src/building-blocs/PageHeader.tsx:33]: What tool do you use to view those? 👀

Comment (gdostie) [reply packages/website/src/building-blocs/PageHeader.tsx:33]: The same problem exists on master though, so out of scope for this PR

**Review by GermainBergeron — APPROVED** [review]

File: packages/website/src/building-blocs/PageHeader.tsx (hunk 1 above)
Comment (toofff) [reply packages/website/src/building-blocs/PageHeader.tsx:33]: See this chrome plugin https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en

Out of scope ok, but would it be possible for you to write a stain to fix this?

**Pull request merged**